src/AffectedUserFlow/exposureNotificationAPI.spec.ts

```diff
@@ -109,6 +113,81 @@ describe("postDiagnosisKeys", () => {
     })
   })
 
+  describe("on a retry response", () => {
+    it("retries the request and returns the response of the next not retry", async () => {
```

Comment:
Is this correct or is there a typo? ➡️ `not retry`

Reply:
I think i missed the response at the end 😄